packages/icons/icons.json

```diff
@@ -4198,5 +4198,12 @@
         "tags": "data, type, format，punctuation, grammar",
         "group": "miscellaneous",
         "codepoint": 62296
+    },
+    {
+        "displayName": "Server",
+        "iconName": "server",
```

Comment:
We should prefix these with cloud to match the rest 
<img width="839" alt="image" src="https://github.com/palantir/blueprint/assets/1143755/b5003ee6-f939-4910-b1c9-47fe7746e786">

```suggestion
        "displayName": "Cloud: Server",
        "iconName": "cloud-server",
```